### DISCUSSION TITLE
CopilotChat doesnt work - Expected value but found invalid token at character 1

### DISCUSSION THREAD

**Discussion by ddfonseca** [LazyVim/LazyVim#3646] [Q&A] [answered]
Hi everyone,

I am new to lazyvim and neovim community. I installed through lazyvim extras, so I thought it should be ok to asking here

Basically it says:

[CopilotChat.nvim] "Expected value but found invalid token at character 1"

Copilot is working fine, jus CopilotChat that isn't.

What should I do? Which informations should I give to you?

**Comment by folke** [comment, marked as answer]
I have no idea. It's working for me.
Are you sure copilot chat is enabled on your github account? Not sure that's for everyone already.

**Reply by ddfonseca** [reply]
It was working on VSCode, that why I thought that should work on Neovim. But you make me think twice. Indeed, the company has disabled copilot chat.

Thank your for answering me.
